charts/opentelemetry-demo/templates/_pod.tpl

```diff
@@ -91,7 +91,7 @@ Get Pod Env
 
 {{- if eq .name "product-catalog-service" }}
 - name: FEATURE_FLAG_GRPC_SERVICE_ADDR
-  value: {{ (printf "%s-featureflag-service:%0.f" $prefix .servicePort ) }}
+  value: {{ (printf "%s-featureflag-service:50031" $prefix ) }}
```

Comment:
Logging this hard coded value as a temporary fix.  Will address as part of https://github.com/open-telemetry/opentelemetry-helm-charts/issues/340